changelogs/fragments/4351-inventory-lxd-handling_metadata_wo_os_and_release.yml

```diff
@@ -0,0 +1,6 @@
+bugfixes:
+  - |
+    lxd inventory plugin - fix handling of metadata to be without os and release.
+    This happens if a empty container is created via API.
+    ```lxc query -X POST /1.0/containers -d '{"name": "container-name", "source": {"type": "none"}, "config": {"security.privileged": "true"}}'```
+    (https://github.com/ansible-collections/community.general/pull/4351)
```

Comment:
```suggestion
    lxd inventory plugin - do not crash if OS and release metadata are not present
    (https://github.com/ansible-collections/community.general/pull/4351).
```

Reply:
Your suggested fragment is more concise than mine. I like it